react/features/notifications/reducer.ts

```diff
@@ -90,30 +90,41 @@ ReducerRegistry.register<INotificationsState>('features/notifications',
  * queue.
  */
 function _insertNotificationByPriority(notifications: INotification[], notification: INotification) {
+
+    // Create a copy to avoid mutation.
+    const copyOfNotifications = notifications.slice();
+
+    // Check if any queued notification has the same id as the new notification
+    const sameUIDNotification = (queuedNotification: INotification) => queuedNotification?.uid === notification?.uid;
```

Comment:
Doesn't it look cleaner to inline this in the findIndex call? No big deal though.